### ISSUE TITLE
Window is not defined (SSR support)

### ISSUE THREAD

**Issue by joshferrell** [benbowes/react-responsive-select#72]
This implementation currently uses the window object which is not available when server side rendering. [I found these usages of the window object](https://github.com/benbowes/react-responsive-select/search?q=window&unscoped_q=window). Is it possible that this could add support for server side rendering?

**Comment by benbowes** [comment]
It can be used server side, I am using it SSR

**Comment by benbowes** [comment]
@joshferrell could you please supply the error you are getting?

**Comment by joshferrell** [comment]
```
 [ 'ReferenceError: window is not defined',
[3]   'at Object.<anonymous> (FILE_PATH/node_modules/react-responsive-select/dist/ReactResponsiveSelect.
js:1:286)',
```

**Comment by benbowes** [comment]
Cheers @joshferrell,

I will investigate today.

**Comment by benbowes** [comment]
Yep. I see the problem and will fix asap.

Thanks for letting me know.

**Comment by benbowes** [comment]
This appears to be a Webpack export issue.

In https://github.com/benbowes/react-responsive-select/blob/master/webpack.config.react-responsive-select.js#L24 - changing `umd` to `commonjs2` appears to fix the problem.

However I need to test this a bit more before committing to the change.

**Comment by benbowes** [comment]
@joshferrell should get fixed in https://github.com/benbowes/react-responsive-select/pull/74

**Comment by benbowes** [comment]
Released react-responsive-select@4.0.1
